.pre-commit-config.yaml

```diff
@@ -9,6 +9,10 @@ repos:
       - id: check-yaml
       - id: check-added-large-files
       - id: check-merge-conflict
+  - repo: https://github.com/pre-commit/mirrors-autopep8
+    rev: 898691a48e7dbdbbac32f44f9336ef944d74cf82
```

Comment:
Any tag maybe to use here?

Reply:
nope, c.f. README on the hook repo. Need to use a SHA unfortunately

Reply:
Misread the README, you are right. Using v1.6.0 tag